core/src/main/java/org/apache/accumulo/core/iterators/ColumnFamilyCounter.java

```diff
@@ -89,4 +89,11 @@ public Value getTopValue() {
     return null;
   }
 
+  @Override
```

Comment:
I wonder if we could clean up some of these iterators implementations by making them use `WrappingIterator`.